src/helpers/tests/measureTitleHelper.spec.ts

```diff
@@ -205,4 +205,64 @@ describe("measureTitleHelper", () => {
             ).toBeUndefined();
         });
     });
+
+    describe("ignoreTitles", () => {
+        function getTitleOfMeasure(
+            visualizationObject: IVisualizationObjectContent,
+            localIdentifier: string,
+        ): string {
+            const measureBucketItems = visualizationObject.buckets[0].items;
+            const matchingMeasure: IMeasure = measureBucketItems
+                .map(bucketItem => bucketItem as IMeasure)
```

Comment:
please add type and return type for all arrow functions :)

Reply:
Fixed!